optical/README.md

```diff
@@ -240,10 +240,10 @@ For those details please refer to the ["Additional properties" column in the tab
 | Field Name      | Req.  | Data Type                                                    | Description                                                  |
 | --------------- | ----- | ------------------------------------------------------------ | ------------------------------------------------------------ |
 | data_type       | *n/a* | string                                                       | One of the [Data Types](https://github.com/stac-extensions/raster/blob/v1.1.0/README.md#data-types). |
+| bits_per_sample | *n/a* | integer                                                      | Actual number of bits per sample (e.g., 8, 16, 32, ...) if the number of bits is non-standard for the `data_type`. |
```

Comment:
I think Card4l requires this to be specifically set, not only if it is non-standard...

Reply:
```suggestion
| bits_per_sample | *n/a* | integer                                                      | Actual number of bits per sample (e.g., 8, 16, 32, ...) |
```